### ISSUE TITLE
Erroneous Title of resource: " SciSmalltalk"

### ISSUE THREAD

**Issue by jigyasa-grover** [jig08/Pharo-Smalltalk-Outreach-Resources#1]
The title of the [resource](https://github.com/jig08/Pharo-Smalltalk-Outreach-Resources/blob/master/SciSmalltalk.pdf) has been erroneously put as _SciSmalltalk_ instead of _Kendrick_

Thanks @SergeStinckwich for pointing that out.

**Comment by SergeStinckwich** [comment]
Yes information about Kendrick are available here: http://ummisco.github.io/kendrick/
